src/componentes/mounted/busca-ativa/citopatologico/APS/TabelaAPSSemExame.jsx

```
@@ -31,126 +31,87 @@ const IDFiltrosOrdenacaoCito = {
+    tabelaData,
+    setTabelaData,
+    mixpanel,
+    aba,
+    sub_aba,
+    showSnackBar,
+    setShowSnackBar,
+    setFiltros_aplicados,
+    liberarPesquisa
+})=>{
+    const tabelaDataAPSSemExame = tabelaDataAPS?.filter(item=>item.id_status_usuario !== 12)
+    .map(item => ({...item, equipe_nome_e_ine: `${item.equipe_nome} - ${item.equipe_ine}`}))
```

Biome (1.9.4)

[error] This object property name should be in camelCase.



(lint/style/useNamingConvention)